components/builder-web/stylesheets/base/_forms.scss

```diff
@@ -136,3 +136,15 @@ select {
   -webkit-appearance: none;
   width: auto;
 }
+
+::placeholder {
+  color: $hab-gray;
+}
```

Comment:
Changes global placeholder text to gray instead of dark blue.

Reply:
Thank you!